apiserver/params/internal.go

```diff
@@ -83,18 +64,6 @@ type MachinePorts struct {
 	NetworkTag string
 }
 
-// MachinePortRange holds a single port range open on a machine for
-// the given unit and relation tags.
-type MachinePortRange struct {
-	UnitTag     string
-	RelationTag string
-	// TODO(dimitern): Add explicit JSON serialization tags and use
-	// string instead in order to break the dependency on the network
-	// package, as this potentially introduces hard to catch and debug
-	// wire-format changes in the protocol when the type changes!
-	PortRange network.PortRange
-}
-
 // MachinePortsParams holds the arguments for making a
 // FirewallerAPIV1.GetMachinePorts() API call.
 type MachinePortsParams struct {
```

Comment:
and these down below: EntityPort, EntitiesPorts, EntityPortRange, EntitiesPortRanges